next.config.js

```diff
@@ -2,8 +2,17 @@
 const nextConfig = {
   reactStrictMode: true,
   productionBrowserSourceMaps: true,
+  env: {
+    AURORA_AWS_ACCESS_KEY_ID: process.env.AURORA_AWS_ACCESS_KEY_ID,
+    AURORA_AWS_SECRET_ACCESS_KEY: process.env.AURORA_AWS_SECRET_ACCESS_KEY,
+    AURORA_DATABASE: process.env.AURORA_DATABASE,
+    AURORA_RESOURCE_ARN: process.env.AURORA_RESOURCE_ARN,
+    AURORA_SECRET_ARN: process.env.AURORA_SECRET_ARN,
+    SECRET_COOKIE_PASSWORD: process.env.SECRET_COOKIE_PASSWORD,
+    FRONTEND_URL: process.env.FRONTEND_URL,
+  },
```

Comment:
This is already removed. All succeeding environment variable changes should be placed in `amplify.yml`

```suggestion
  },
```